test/testcafe/spec/ChallengeAuthenticatorPhone_spec.js

```diff
@@ -185,6 +198,26 @@ test
     await t.expect(challengePhonePageObject.getSignoutLinkText()).eql('Back to sign in');
   });
 
+test.meta('v3', false)
+  .requestHooks(smsPrimaryNicknameMock)('SMS primary mode - shows nickname when present in API response', async t => {
+    const challengePhonePageObject = await setup(t);
+    await checkA11y(t);
+
+    const pageTitle = challengePhonePageObject.getFormTitle();
+    const pageSubtitle = challengePhonePageObject.getFormSubtitle();
+    const primaryButtonText = challengePhonePageObject.getSaveButtonLabel();
+    const secondaryButtonText = challengePhonePageObject.getSecondaryLinkText('Receive a voice call instead');
+    await t.expect(pageTitle).contains('Verify with your phone');
+    await t.expect(pageSubtitle).contains('Send a code via SMS to');
+    await t.expect(pageSubtitle).contains('+1 XXX-XXX-2342');
+    await t.expect(pageSubtitle).contains(' (phone-nn)');
+    await t.expect(primaryButtonText).contains('Receive a code via SMS');
+    await t.expect(secondaryButtonText).contains('Receive a voice call instead');
```

Comment:
These assertions are not needed as they are for a different view.
```suggestion
```

Reply:
Addressed.